src/config/config.js

```diff
@@ -113,6 +114,11 @@ const Config = {
   torusEmailEnabled: env.REACT_APP_TORUS_AUTH0EMAIL_ENABLED === 'true',
   torusUxMode: isWeb ? (env.REACT_APP_TORUS_UXMODE || 'redirect') : 'popup',
   abTestPercentage: env.REACT_APP_AB_TEST_PERCENTAGE || 0.5,
+
+  // TODO: remove feed migration env vars after feed migration is done
+  feedMigrationDate: env.REACT_APP_FEED_MIGRATION_DATE || moment().format('YYYY/MM/DD'),
+  feedMigrationBlock: env.REACT_APP_FEED_MIGRATION_BLOCK || 6000000,
```

Comment:
we dont need it in all the .env or the settings file, just hardcode it in the specific update file

Reply:
I think we need this in case we would like to re-run the migration for any reason we could do it without a hotfix
it is temporary and can be removed after the feed migration is finally over

Reply:
please remove it